compiler/one-cmds/utils.py

```diff
@@ -89,6 +89,15 @@ def _add_default_arg(parser):
         action='store_true',
         help='show program\'s version number and exit')
 
+    # verbose
+    parser.add_argument(
+        '-V',
+        '--verbose',
+        action='store_const',
+        default='0',
```

Comment:
Isn't it `2` for normal case?

Reply:
Plus, how about using `count` action?
```python
parser.add_argument('-V', '--verbose', action='count', default=0)
```
Then, seems better argument parsing.
`-V` -> count 1
`-VV` -> count 2
`-VVV` -> count 3
`` -> count 0
`-V -V` -> count 2
..

Reply:
> Isn't it `2` for normal case?

Good point.  We need to decide.

Q. Is the meaning of verbose level same to `TF_CPP_MIN_LOG_LEVEL`? (0: All, ..., 3: Fatal)

Reply:
> Plus, how about using `count` action?
> 
> ```python
> parser.add_argument('-V', '--verbose', action='count', default=0)
> ```
> 
> Then, seems better argument parsing.
> `-V` -> count 1
> `-VV` -> count 2
> `-VVV` -> count 3
> ` ` -> count 0
> `-V -V` -> count 2
> ..

I would like to provide consistent style for our tool's arguments. Most tools are written in python, however, if we have some tool written in C++, the tool is not likely to provide the handling of `-VVV` and so on.

Reply:
@glistening 
> Q. Is the meaning of verbose level same to `TF_CPP_MIN_LOG_LEVEL`? (0: All, ..., 3: Fatal)

Oh, I misunderstood the codes. Current change looks okay to me.

> I would like to provide consistent style for our tool's arguments. Most tools are written in python, however, if we have some tool written in C++, the tool is not likely to provide the handling of `-VVV` and so on.

Well, okay. I just thought current tools for users is `one-cmds` and it only consists of python scripts. Both ways looks good to me.

Reply:
> Plus, how about using count action?

This is a good suggestion. However our verbose level has not yet been precisely defined, so I want to expand it later. Currently it supports to turn verbose option on and off. Also, additional consideration is needed on how to handle arguments in our multiple type of tools.